Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
```

Comment:
### [concurrent-ruby](http://www.concurrent-ruby.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.1 → 1.1.5

[_~~change-log~~, [source-code](https://github.com/ruby-concurrency/concurrent-ruby)_]





<details>
<summary>Commits</summary>

A change of **110** commits. See the full changes on [the compare page](https://github.com/ruby-concurrency/concurrent-ruby/compare/v1.1.1...v1.1.5).

These are the first 10 commits:
- (cf28431) [Defensive 1.9.3 support](https://github.com/ruby-concurrency/concurrent-ruby/commit/cf28431816435e33e8ef190a17a53295584f2a7c)
- (db8f88b) [Merge pull request #773 from ruby-concurrency/193](https://github.com/ruby-concurrency/concurrent-ruby/commit/db8f88b63a403645eb4f621743edf2453232b150)
- (a18c5d0) [bump version](https://github.com/ruby-concurrency/concurrent-ruby/commit/a18c5d0b07c3453a3fa87fa433098797ea170ea6)
- (9fe401e) [Allow publishing only core](https://github.com/ruby-concurrency/concurrent-ruby/commit/9fe401e14b8de22ca27c4dc4575e382b756d7811)
- (6b7ec84) [Update CHANGELOG.md](https://github.com/ruby-concurrency/concurrent-ruby/commit/6b7ec84d97caa11931d5626ff2a44ca63c7bf54a)
- (7833d9c) [Fix NameError on defining a struct with a name](https://github.com/ruby-concurrency/concurrent-ruby/commit/7833d9cb557236abee58532c18c2dbec62ad741e)
- (b942fcc) [fix partial require of the gem](https://github.com/ruby-concurrency/concurrent-ruby/commit/b942fccf81637721d6d54c055bfa51fb9c58a62a)
- (675c746) [version bump](https://github.com/ruby-concurrency/concurrent-ruby/commit/675c74653ac502a951f8b46acc014db54d0c2ae7)
- (dd6b5f5) [Merge pull request #776 from Slike9/fix-struct-with-class-na…](https://github.com/ruby-concurrency/concurrent-ruby/commit/dd6b5f5740f1c75e9edf26363fc593e883ce4ff2)
- (619da05) [Update CHANGELOG.md](https://github.com/ruby-concurrency/concurrent-ruby/commit/619da05a1883fd73b31b41d419db417eed69fc9d)


</details>